packages/manager/test/SliceMachineManager-versions-checkIsSliceMachineUpdateAvailable.test.ts

```diff
@@ -24,7 +24,7 @@ it("returns true if an update is available", async (ctx) => {
 		versions: ["1.0.0", "2.0.0"],
 	});
 
-	const res = await manager.versions.checkIsUpdateAvailable();
```

Comment:
Should we still have a `checkIsUpdateAvailable()` method available? It could tell very generally if an update is available (checking for an SM UI and adapter update), which may be used in SM UI to display the sidebar update message.

Reply:
I ask myself the same question, even if it could help to have only method, the two methods called within will not change at the end. So I choose two functions, mostly to prevent us to be force to call NPM for the adapters if we don't need the info. Today we only call both of them at the same time, but it's a performance issue if not one day, right?

Let me know if you think it's better, and I could merge the two for sure 🙂

Reply:
Sounds good to me! We can always introduce a convenience method that combines all update checkers if needed in the future. 🙂